src/goto-symex/goto_symex_state.h

```diff
@@ -148,8 +148,7 @@ class goto_symex_statet final : public goto_statet
 
   const call_stackt &call_stack() const
   {
-    PRECONDITION(source.thread_nr < threads.size());
```

Comment:
Whatever happened to this `PRECONDITION`? Why did it disappear? Is it being reintroduced somewhere else? I couldn't find it if so. If not, removing invariants and preconditions is a pretty bad idea, unless you can reason that that the invariant was badly reasoned to begin with.

Reply:
I'm dropping this commit which makes no sense